libs/newsletter-workflow/src/lib/steps/newsletterData/frequencyLayout.ts

```diff
@@ -11,15 +11,16 @@ const markdownTemplate = `
 # Specify the send frequency of {{name}}
 
 Specify how regularly the newsletter will be sent e.g.
-- every day
-- every weekday
-- weekly
-- fortnightly
-- monthly
+- Every day
```

Comment:
These are deliberately lowercase because of where they appear (see screenshots on this page), and the examples should reflect what we want the user to enter

Reply:
Ignore this comment too - it's mixed case in the spreadsheet, so we're obviously lower-casing it before use (which is exactly as it should be)!